apps/frontend/src/App.tsx

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 
 import LandingPage from './pages/LandingPage';
+
 import Registration from './pages/Registration';
+import RegistrationForm from './components/RegistrationForm';
+
 import LoginPage from './pages/LoginPage';
 import LoginForm from './components/LoginForm';
 
 function App() {
   return (
     <Routes>
       <Route path="/" element={<LandingPage />} />
-      <Route path="/register" element={<Registration />} />
+      <Route path="/register" element={<Registration formComponent={<RegistrationForm />} />} />
```

Comment:
Passing `formComponent` prop inside `App.tsx` probably isn't the best approach (i.e. this page shouldn't be aware of the lower-level implementation details of sign-up and login pages).

Reply:
Great point.

I modified the two components so they no longer take forms as props; instead, I simply instantiate the forms within the pages themselves.

<img width="575" alt="Screenshot 2024-03-01 at 09 26 53" src="https://github.com/Geek-Collectors-Network/geek-collectors-network/assets/97256979/a113026d-79da-4280-b0d2-adfe9ce54c51">
<img width="593" alt="Screenshot 2024-03-01 at 09 26 19" src="https://github.com/Geek-Collectors-Network/geek-collectors-network/assets/97256979/7a4ce44b-0ff0-4219-bca7-74f9f692e927">